src/logger/fsbackend.cpp

```diff
@@ -0,0 +1,392 @@
+/*
+ * Copyright (C) 2025 EPAM Systems, Inc.
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+#include <assert.h>
+#include <fcntl.h>
+#include <stdio.h>
+#include <stdlib.h>
+#include <sys/stat.h>
+#include <sys/time.h>
+#include <unistd.h>
+
+#include <zephyr/fs/fs.h>
+#include <zephyr/logging/log_backend.h>
+#include <zephyr/logging/log_backend_std.h>
+#include <zephyr/logging/log_ctrl.h>
+#include <zephyr/logging/log_output_custom.h>
+#include <zephyr/logging/log_output_dict.h>
+#include <zephyr/sys/timeutil.h>
+
+#include <aos/common/tools/fs.hpp>
+#include <aos/common/tools/memory.hpp>
+
+#include "fsbackend.hpp"
+#include "utils/utils.hpp"
+
+namespace aos::zephyr::logger::backend {
+
+namespace {
+
+/***********************************************************************************************************************
+ * Static
+ **********************************************************************************************************************/
+
+uint32_t sCurrentLogFormat = 0;
+
+constexpr auto cLogFlags = LOG_OUTPUT_FLAG_LEVEL | LOG_OUTPUT_FLAG_TIMESTAMP | LOG_OUTPUT_FLAG_FORMAT_TIMESTAMP;
+
+const struct log_backend* log_backend_aos_get(void);
+
+void ActivateBackend()
+{
+    log_backend_activate(log_backend_aos_get(), nullptr);
+}
+
+void DeactivateBackend()
+{
+    log_backend_deactivate(log_backend_aos_get());
+}
+
+int UTCTimestamp(
+    const struct log_output* output, const log_timestamp_t timestamp, const log_timestamp_printer_t printer)
+{
+    auto now = Time::Now();
+
+    auto [utcTimeStr, err] = now.Time::ToUTCString();
+    if (!err.IsNone()) {
+        return printer(output, "%s", utils::ErrorToCStr(err));
+    }
+
+    return printer(output, "%s", utcTimeStr.CStr());
+}
+
+int HandleLog(const uint8_t* data, size_t length, void* ctx)
+{
+    return FSBackend::Get().HandleLog(data, length);
+}
+
+uint8_t __aligned(4) sLogBuffer[cLogEntryLen];
+LOG_OUTPUT_DEFINE(sLogOutput, reinterpret_cast<log_output_func_t>(HandleLog), sLogBuffer, cLogEntryLen);
+
+void Panic(struct log_backend const* const backend)
+{
+    DeactivateBackend();
+}
+
+void Dropped(const struct log_backend* const backend, uint32_t cnt)
+{
+    ARG_UNUSED(backend);
+
+    if (IS_ENABLED(CONFIG_LOG_BACKEND_FS_OUTPUT_DICTIONARY)) {
+        log_dict_output_dropped_process(&sLogOutput, cnt);
+    } else {
+        log_backend_std_dropped(&sLogOutput, cnt);
+    }
+}
+
+void Process(const struct log_backend* const backend, union log_msg_generic* msg)
+{
+    log_format_func_t_get(sCurrentLogFormat)(&sLogOutput, &msg->log, cLogFlags);
+}
+
+int SetFormat(const struct log_backend* const backend, uint32_t log_type)
+{
+    sCurrentLogFormat = log_type;
+
+    return 0;
+}
+
+const struct log_backend_api aos_log_backend_fs_api = {
```

Comment:
We should follow zephyr naming. It should be:

log_backend_aos..
or 
log_backend_aos_fs...